app/uk/gov/hmrc/apinotificationpull/controllers/NotificationsController.scala

```diff
@@ -32,8 +35,9 @@ import scala.concurrent.Future
 
 @Singleton
 class NotificationsController @Inject()(apiNotificationQueueService: ApiNotificationQueueService,
-                                        headerValidator: HeaderValidator) extends BaseController {
-
+                                        headerValidator: HeaderValidator,
+                                        apiNotificationQueueConnector: ApiNotificationQueueConnector,
```

Comment:
I suggest you remove the dependency to the connector.
The controller talks to the service.
The service talks to the connector.

Reply:
I intend to refactor the logic currently in the delete method into the service, at which point this dependency will be removed.